pkg/tempofb/tempo.fbs

```diff
@@ -0,0 +1,24 @@
+namespace tempofb;
+
+table KeyValues {
+    key: string;
+    value: [string];
+}
+
+table SearchData {
+    id : string; // Converted to []byte
+    tags : [KeyValues];
+    start_time_unix_nano: uint64;
+    end_time_unix_nano: uint64;
+}
+
+table BatchSearchData {
```

Comment:
A little confused by the naming, should we make this `BlockSearchData`?

Reply:
A batch is actually a page.  How about BatchSearchData -> SearchPage, SearchData -> SearchEntry ?

Reply:
Yup these two renames make sense to me.